week-1-ts-ecosystem/w01e05-my-own-cli/src/demo-prepare/actions/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { DataCollection } from './types';
+import fs from 'fs';
+import path from 'path';
+
+export const Data = {
+  read: (): DataCollection => {
+    const rawData = fs.readFileSync(
+      path.resolve(process.cwd(), 'persistent-data/cart-items.json'),
+      'utf-8'
+    );
+
+    return JSON.parse(rawData);
+  },
+  write: (data: DataCollection): void => {
+    fs.writeFileSync(
+      path.resolve(process.cwd(), 'persistent-data/cart-items.json'),
+      JSON.stringify(data)
+    );
+  },
+};
```

Comment:
Jakoś inaczej bym to nazwał jako plik.

być może nazwa `utils` na początku Ci pasowała.
Ale teraz wyraźnie widać że to jest obsługa zapisu do `pesistent` i takim kluczem bym się suegerował w nazwie.

Dodatkowo nie nazywałbym `Data` z dużej - bo to sugeruje że to klasa.
A tutaj mamy wyraźnie instancję realizującą zadania.

Nie wiem skąd u Ciebie takie zamiłowanie do `=>` arrow :) w tym projekcie
wydaje mi się ze dużo prościej by było: 

```typescript
read(): DataCollection { ...
...
write(data: DataCollection): void {
```

Reply:
No i nazwa "data" - niewiele mówi...